ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/DBFetchingProcessRepository.scala

```diff
@@ -94,13 +94,15 @@ abstract class DBFetchingProcessRepository[F[_]: Monad](val dbConfig: DbConfig)
   private def fetchProcessDetailsByQueryAction[PS: ProcessShapeFetchStrategy](query: ProcessEntityFactory#ProcessEntity => Rep[Boolean],
                                                                               isDeployed: Option[Boolean])(implicit loggedUser: LoggedUser, ec: ExecutionContext): DBIOAction[List[BaseProcessDetails[PS]], NoStream, Effect.All with Effect.Read] = {
     (for {
-      deployments <- fetchLastDeploymentActionPerProcess.result
+      deployments <- fetchLastActionPerProcess.result
```

Comment:
It is not `deployments` anymore, but `lastAction`, you can also change `latestDeployed` to `lastDeployedAction` below.